### PR TITLE
certificate-authentication.md: refine cert based authentication

### PR DESCRIPTION
**Pull request by yikeke**
### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/3328, https://github.com/pingcap/docs-cn/pull/3429
- Other reference link(s):

### CODE REVIEW

**Comment by TomShawn** [comment]
LGTM
@lysu PTAL

**Comment by yikeke** [comment]
I aligned another doc, PTAL again @TomShawn @lysu

File: certificate-authentication.md [hunk 1]
```diff
@@ -287,6 +279,32 @@ The user certificate information can be specified by `require subject`, `require
...
+        openssl x509 -noout -ext subjectAltName -in client.crt
+        ```
+
+    + `require san` currently supports the following `Subject Alternative Name` check items:
+
+        - URI
+        - IP
+        - DNS
+
+    + Multiple check items can be configured after connecting by commas. For example, configure as follows for the `u1` user:
```
Comment (TomShawn) [inline certificate-authentication.md:298]:
```suggestion
    + Multiple check items can be configured after connected by commas. For example, configure `require san` as follows for the `u1` user:
```

**Pull request merged**